### ISSUE TITLE
Select Datasets from UCR files

### ISSUE THREAD

**Issue by nssafir** [quinngroup/CiliaRepresentation#48]
Pick which datasets from all the UCR files we should use to train dynamics head. It should be similar in terms of the number of points in the path and dimensionality of the points.

**Comment by nssafir** [comment]
Is there a range for number of points for path and dimensionality that we're looking for?

**Comment by Micky774** [comment]
Not necessarily. Dimensional would be ideally between 2-100 but it's all very arbitrary.
